### PR TITLE
Remove expected file checks

### PR DESCRIPTION
**Pull request by shrutir25**
Deleted expected template and parameter files
Removed generation and comparing of template and param files with expected ones.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/547)

### CODE REVIEW

**Comment by shrutir25** [comment]
@acs-bot test this please

**Comment by shrutir25** [comment]
@anhowe could you review this ?

File: pkg/acsengine/engine_test.go [hunk 1]
```diff
@@ -66,20 +51,44 @@ func TestExpected(t *testing.T) {
...
 			continue
 		}
+
+		armTemplate, params, certsGenerated, err := templateGenerator.GenerateTemplate(containerService)
+		if err != nil {
+			t.Error(fmt.Errorf("error in file %s: %s", tuple.APIModelFilename, err.Error()))
+			continue
+		}
+
+		ppArmTemplate, e1 := PrettyPrintArmTemplate(armTemplate)
```
Comment (anhowe) [inline pkg/acsengine/engine_test.go]: for ease of reading prefix ppArmTemplate and ppParams with "expected" or "target"

Comment (shrutir25) [inline pkg/acsengine/engine_test.go]: I have applied 'expected' and 'generated' prefix accordingly for ppArmTemplate and ppParams

**Comment by shrutir25** [comment]
@acs-bot test this please

**Comment by dmitsh** [comment]
@acs-bot test this please

**Pull request merged**